docs/docs-ref-autogen/outlook/office.categories.yml

```diff
@@ -109,99 +109,22 @@ items:
 
       ```javascript
 
-      var masterCategories = [
-          {
-              "displayName": "Urgent!",
-              "color": Office.MailboxEnums.CategoryColor.Preset0
-          }
-      ];
-
-
-      Office.context.mailbox.masterCategories.getAsync(getMasterCategoriesCallback);
-
-
-      Office.context.mailbox.masterCategories.addAsync(masterCategories, function (asyncResult) {
-          if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
-              console.log("Successfully added categories to master list");
-          } else {
-              console.log("masterCategories.addAsync call failed with error: " + asyncResult.error.message);
-          }
-      });
-
-
-      Office.context.mailbox.masterCategories.getAsync(getMasterCategoriesCallback);
+      // Note: In order for you to successfully add a category,
 
```

Comment:
Nit: Let's remove the spaces between the line comments.

Reply:
I think that's just due to the tool. This is how it looks on [Office.Categories](https://review.docs.microsoft.com/en-us/javascript/api/outlook/office.categories?view=office-js&branch=elizs-add-categories-samples#addasync-categories--callback-):
![image](https://user-images.githubusercontent.com/28541963/56381880-ee407300-61ca-11e9-8884-d292f111db1d.png)